packages/@okta/vuepress-site/docs/reference/api/schemas/index.md

```diff
@@ -1228,61 +1212,61 @@ Custom property names for the [profile object](/docs/reference/api/users/#profil
-| lastName          | family name of the user (`familyName`)                                                                                             | String   | FALSE (default)    | FALSE  | FALSE    | 1         | 50        |                                                                                                                   |
-| middleName        | middle name(s) of the user                                                                                                         | String   | FALSE    | FALSE  | FALSE    |           |           |                                                                                                                   |
-| honorificPrefix   | honorific prefix(es) of the user, or title in most Western languages                                                               | String   | TRUE     | FALSE  | FALSE    |           |           |                                                                                                                   |
-| honorificSuffix   | honorific suffix(es) of the user                                                                                                   | String   | TRUE     | FALSE  | FALSE    |           |           |                                                                                                                   |
-| title             | user's title, such as "Vice President"                                                                                       | String   | TRUE     | FALSE  | FALSE    |           |           |                                                                                                                   |
-| displayName       | name of the user, suitable for display to end users                                                                                | String   | TRUE     | FALSE  | FALSE    |           |           |                                                                                                                   |
-| nickName          | casual way to address the user in real life                                                                                        | String   | TRUE     | FALSE  | FALSE    |           |           |                                                                                                                   |
-| profileUrl        | url of user's online profile (e.g. a web page)                                                                               | String   | TRUE     | FALSE  | FALSE    |           |           | [URL](https://tools.ietf.org/html/rfc1808)                                                                        |
```

Comment:
"URL"?

Reply:
good catch!

Reply:
this actually WAS correct - but super confusing...looks like there's something missing - but it's going to the spec for URLs...so I spelled it out to limit confusion.